<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PPIVisualSettings</class>
 <widget class="QWidget" name="PPIVisualSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PPI视觉设置</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="distanceLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_distance">
       <property name="text">
        <string>最大距离(km)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxDistanceEdit">
       <property name="text">
        <string>500</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mapLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_map">
       <property name="text">
        <string>地图类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomComboBox" name="mapTypeCombo">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>无地图</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>路网图</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>标准图</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>卫星图</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="measureLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="measureBtn">
       <property name="text">
        <string>测距</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/resources/icon/meas.png</normaloff>
        </iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>测量模式</string>
       </property>
       <property name="objectName">
        <string>MeasureBtn</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>cusWidgets/customcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>